pkg/file/kong_json_schema.json

```diff
@@ -1500,6 +1600,30 @@
       "additionalProperties": false,
       "type": "object"
     },
+    "KeySet": {
```

Comment:
What is the difference between `FKeySet` and `KeySet`?

Reply:
FKeySet is the representation of a keyset in a file. We will use it for parsing and writing the keyset object.

KeySet is defined for the reference used here in keys: https://github.com/Kong/go-database-reconciler/blob/feat/file-parse-keys-keysets/pkg/file/kong_json_schema.json#L736
For embedded references, we don't use the F* prefix to avoid unnecessary loops and also to comply with the go object structure.